dist/skeleton-loader/skeleton-loader.css

```diff
@@ -0,0 +1,186 @@
+:root {
+  --skeleton-border-radius: 0;
+  --skeleton-height: 48px;
+  --skeleton-tile-width-max: 224px;
+  --skeleton-tile-width-min: 121px;
+  --skeleton-width: 100%;
+}
+.skeleton {
+  background: var(--color-loading-fill);
+  border-radius: var(--skeleton-border-radius);
+  cursor: progress;
+  height: var(--skeleton-height);
+  min-height: 12px;
+  width: var(--skeleton-width);
+}
+.skeleton-elevated {
```

Comment:
What is skeleton-elevated?

Reply:
@ianmcburnie I believe it’s for skeleton elements on "elevated" surfaces like modals and headers

Reply:
[DS](https://www.figma.com/file/zEBdEhbonrBOGzZ0fXzWvM/eBay-Design-System?type=design&node-id=91944-98600&mode=design&t=SO5ouM1SoRxy1AvA-0) has a different token especially on dark mode. I guess this is needed to show contrast on dark mode.